### PR TITLE
LPS-151531 | LPS-151532

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issues: 
https://issues.liferay.com/browse/LPS-151531
https://issues.liferay.com/browse/LPS-151532

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/data/set/view/table/ObjectActionsTableFDSView.java [hunk 1]
```diff
@@ -47,6 +47,9 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 
 		nameFDSTableSchemaField.setContentRenderer("actionLink");
 
+		fdsTableSchemaBuilder.addFDSTableSchemaField(
```
Comment (rodrigopaulino) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/data/set/view/table/ObjectActionsTableFDSView.java]: Can you sort this alphabetically?

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsActionsDisplayContext.java [hunk 2]
```diff
@@ -190,6 +191,24 @@ public JSONArray getObjectActionExecutorsJSONArray() {
...
+		).put(
+			"description", objectAction.getDescription()
+		).put(
+			"id", objectAction.getObjectActionId()
+		).put(
+			"name", objectAction.getName()
+		).put(
+			"objectActionExecutorKey", objectAction.getObjectActionExecutorKey()
+		).put(
+			"objectActionTriggerrKey", objectAction.getObjectActionTriggerKey()
```
Comment (rodrigopaulino) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsActionsDisplayContext.java:206]: There's an extra "R" there. 😛

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hey, @carolmariaabb. Nice work. 👏🏻 

I've made just a couple of notes below, but, besides them, don't you think the new description  attribute should be introduced under feature flag as well? I understand that it's not showing in UI by default, but API users are able to save and get this information.

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/data/set/view/table/ObjectActionsTableFDSView.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/data/set/view/table/ObjectActionsTableFDSView.java]: Done

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsActionsDisplayContext.java (hunk 2 above)
Comment (carolmariaabb) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsActionsDisplayContext.java]: Fixed! Thanks 😄

**Comment by carolmariaabb** [comment]
> don't you think the new description attribute should be introduced under feature flag as well?

@rodrigopaulino please check the commit [a9f62efda](https://github.com/liferay-forms/liferay-portal/pull/1597/commits/a9f62efdaf186df9ae72268a35ecd9b2230c658b). 

It was the only way I could think of how to put this attribute under FF.

Thanks!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by rodrigopaulino — COMMENTED** [review]
As discussed offline, we're going to send the action's description changes with the new page refactoring, therefore, they won't need feature flag. For now, we'll just request pulls related to the groovy script execution.

cc @carolmariaabb

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:reevaluate:1387401_16

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**